### PR TITLE
First Update of Wiki & Update Ben/Aaron's Page

### PR DESCRIPTION
**Pull request by ryanstwrt**
Sorry this is a big pull request.

I've updated the wiki and created a simple format. 
Currently not much is there, but it should be a better placeholder.

Along with that I have updated Ben/Aaron's page as well.

### CODE REVIEW

File: users/tamashia/index.md [hunk 1]
```diff
@@ -8,93 +8,7 @@ Aaron Tamashiro
...
-### Skills
-* **Programing Languages:** MATLAB, Python, C/C++/ROOT, and Latex
-* **Tools and Utilities:** MCNP, Google Sketch up, Solid works, Microsoft Word, Microsoft Excel, Microsoft Powerpoint, and Adobe Photoshop
-* **Skills**: Circuits, Vacuum systems, Oscilloscope, High Voltage, and Liquid Nitrogen.
-
-***
-
-### Personal interests
-Cars, lifting, and reading.
+Ben Pharn graduated from Oregon State University in 2019 with a Master's in Nuclear Science and Engineering.
```
Comment (albeanth) [inline users/tamashia/index.md]: ~Ben Pharn~, replace with Aaron Tamashiro.

File: users/tamashia/index.md [hunk 2]
```diff
@@ -8,93 +8,7 @@ Aaron Tamashiro
...
+He is currently still at Oregon State University working under Dr. Camille Palmer.
```
Comment (albeanth) [inline users/tamashia/index.md]: 1. Flip lines 12 and 14? Helps with continuity.
2. Slight edit for clarity:
He is currently ~still~ pursuing a Ph.D. at Oregon State University working under Dr. Camille Palmer.

Comment (ryanstwrt) [reply users/tamashia/index.md]: Thanks Tony! I'll get those updated and pushed through!

**Review by nickwhitman1993 — APPROVED** [review]
@ryanstwrt Looks good to me. Good job man!
EDIT: Well @albeanth found another small one.

**Comment by albeanth** [comment]
@rtrp/admins I still have permissions to merge PRs. Is this by design?

**Comment by nickwhitman1993** [comment]
> @rtrp/admins I still have permissions to merge PRs. Is this by design?

Shouldn't be...

**Comment by albeanth** [comment]
@harterj are you able to merge this PR?

**Comment by nickwhitman1993** [comment]
@albeanth is it possible that since I approved it/you were assigned as a reviewer?

**Comment by albeanth** [comment]
@nickwhitman1993 I think it's because I am still an "owner" of the rtrp organization. 

Myself, Jackson, and Doug are listed as the owners.
https://github.com/orgs/rtrp/people

**Comment by ryanstwrt** [comment]
@nickwhitman1993 @aaronjamesreynolds, if you all are good, I'll merge the pull request.

**Comment by aaronjamesreynolds** [comment]
So, this is what I'm seeing on the wiki page:

image

@ryanstwrt Is the plan to eventually link to the jupyter notebook stuff you've been working on?

**Comment by ryanstwrt** [comment]
@aaronjamesreynolds Correct. I've got the page I've shown on the slack page on my personal page. At some point I can add it to the main wiki as well. I figured we could have something along the lines of :

Ryan Stewart
*Title of work* - link to work

Aaron Reynolds
*Title of work* - link to work

It seems like it would be a neat way to showcase our work.

File: wiki/index.md [hunk 3]
```diff
@@ -1,94 +1,27 @@
...
-* [Unix Shell](http://swcarpentry.github.io/shell-novice/)
-* [C++](http://www.learncpp.com/)
-* [Python3](https://docs.python.org/3/), [Python2](https://docs.python.org/2/)
-* [MATLAB](http://www.mathworks.com/help/matlab/)
-  - [FREE - Matlab’s Online Training Courses](https://matlabacademy.mathworks.com/)
-* [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) - what this website was written in ::
-* And most importantly.... [Google](https://www.google.com/)
-  -  Leads to StackExchange mostly. *Can* be a great reference and a cool place to ask the programming community questions.
+Welcome!
+This wiki page is for current or future student who are interested in Dr. Palmer's research group.
```
Comment (aaronjamesreynolds) [inline wiki/index.md]: This wiki page is for current or future ~~student~~ students

File: users/brancokm/index.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Michael-personal
+layout: default
+author: Michael Branco-Katcher
+---
+# Michael Branco-Katcher
+--------------
+
+I am a junior majoring in nuclear engineering at Oregon State University. My current research involves examening molten salt reactors using the code Moltres.
```
Comment (aaronjamesreynolds) [inline users/brancokm/index.md]: involves ~~examening~~ examining molten salt

File: wiki/index.md (hunk 3 above)
Comment (ryanstwrt) [reply wiki/index.md]: Thanks! I've updated and added an example of what the interactive media will look like.

File: users/pharnb/index.md [hunk 5]
```diff
@@ -6,29 +6,6 @@ author: Ben Pharn
...
-  * Local Student Section Webmaster September 2015 - June 2016
-
-* Institute of Nuclear Materials Management: September 2015 - Present
-  * Member
-
-## Personal Interests
-
-* I like to read in my spare time, and generally listen to music whenever I am doing work.  I am also an avid player of Dungeons and Dragons and other tabletop games.  
\ No newline at end of file
+His thesis work it title *Incorporating Statistical Uncertainty into Nuclear Material Diversion Pathway Analysis using Probabilistic Risk Assessment* and can be found [here](https://ir.library.oregonstate.edu/concern/graduate_thesis_or_dissertations/cn69m9553).
```
Comment (aaronjamesreynolds) [inline users/pharnb/index.md]: His thesis work ~~it title~~ is titled

Comment (ryanstwrt) [reply users/pharnb/index.md]: That should already be updated...

Comment (aaronjamesreynolds) [reply users/pharnb/index.md]: Huh, weird...

**Comment by aaronjamesreynolds** [comment]
Thanks for doing all this @ryanstwrt, it looks real good.

File: users/pharnb/index.md (hunk 5 above)
Comment (ryanstwrt) [reply users/pharnb/index.md]: It says that you are looking at an `outdated` version up top

**Review by aaronjamesreynolds — APPROVED** [review]
Only some minor typos, looks good.

**Comment by ryanstwrt** [comment]
Alrighty, I'll merge this in.

**Pull request merged**